issue-wanted.cabal

```diff
@@ -34,23 +34,82 @@ common common-options
                        -Wpartial-fields
 
   default-language:    Haskell2010
-  default-extensions:  DeriveGeneric
+  default-extensions:  BangPatterns
+                       ConstraintKinds
+                       DataKinds
+                       DeriveGeneric
+                       DerivingStrategies
+                       DerivingVia
+                       FlexibleContexts
+                       FlexibleInstances
                        GeneralizedNewtypeDeriving
+                       InstanceSigs
+                       KindSignatures
                        LambdaCase
+                       MultiParamTypeClasses
                        OverloadedStrings
+                       OverloadedLabels
                        RecordWildCards
                        ScopedTypeVariables
                        TypeApplications
+                       TypeFamilies
+                       TypeOperators
 
 library
   import:              common-options
   hs-source-dirs:      src
   exposed-modules:     Prelude
+
                        IW
-                       IW.Search
-  build-depends:       github == 0.20 
+                         -- Application Monad
+                         IW.App
+                           IW.App.Env
+                           IW.App.Error
+                           IW.App.Monad
+                        
+                         IW.Config
+                         
+                         -- Database
+                         IW.Db
+                           IW.Db.Functions
+                           IW.Db.Schema
+                      
+                         -- Effects
+                         IW.Effects.Log  
+                         
+                         -- Server
+                         IW.Server
+                           IW.Server.Search
+  build-depends:       aeson >= 1.4
+                     , bcrypt ^>= 0.0.11
```

Comment:
We also don't need `bcrypt`. I would propose to try to remove redundant dependencies to make the build faster. Let's not introduce libraries in our `build-depends` unless we use them.

For now it's okay to keep `servant` and `postgresql-simple` dependencies and reexports from `Prelude` for those libraries. Those two will be brought here very soon, so it's okay to have them for now. But other redudant libraries look redundant indeed (and don't forget to remove them from `stack.yaml` as well).